src/queries/queries.js

```diff
@@ -176,31 +174,22 @@ export const buildOneYearOldTimeseriesWithAggregationByAccountId =
     }
   }
 
-export const buildOneYearBikeCommuteTimeseriesQueryByDateAndAccountId = (
-  { date, accountId },
+export const buildBikeCommuteTimeseriesQueryByAccountId = (
+  { accountId },
   enabled
 ) => {
-  const startYearDate = startOfYear(date)
-  const endYearDate = endOfYear(date)
-
   return {
     definition: Q('io.cozy.timeseries.geojson')
       .where({
         'cozyMetadata.sourceAccount': accountId,
-        startDate: {
-          $gte: startYearDate.toISOString(),
-          $lte: endYearDate.toISOString()
-        },
+        startDate: { $gt: null },
         'aggregation.purpose': 'COMMUTE',
-        'aggregation.modes': {
-          $elemMatch: {
-            $eq: BICYCLING_MODE
-          }
-        }
+        'aggregation.modes': { $elemMatch: { $eq: BICYCLING_MODE } }
       })
       .select([
         'startDate',
         'endDate',
+        'aggregation',
```

Comment:
Why is this line needed?
(side note: the commit message has some residuals at the end)

Reply:
Pour avoir le contenu complet de l'agrégation. Select fait en sorte qu'on ne récupère pas l'object au complet, mais seulement un partie. Mais il faut TOUTE la partie aggregation pour que l'app fonctionne.

Reply:
And the `aggregation.*` are there because of https://github.com/cozy/cozy-client/issues/1260